assembly/src/assembler/mod.rs

```diff
@@ -264,6 +264,13 @@ impl Assembler {
             }
         }
 
+        // log the completion of the module
+        #[cfg(feature = "std")]
+        log::debug!(
+            "\n - Module \"{}\" has been compiled",
+            path.unwrap_or(&LibraryPath::anon_path())
+        );
```

Comment:
nit: It would be nice to add a timer, and then I'd probably change the message to "Compiled xyz module in 1.23 ms"

Also, I wonder if we should use `trace!` rather than `debug!` here.